pyexcelerate/Range.py

```diff
@@ -8,6 +11,25 @@
 # to be immutable. Please don't modify attributes after instantiation. :)
 #
 
+# generate the list of columns name from "A" to "ZZZ" => mapping such that COORD2COLUMN[1] => "A"
+COORD2COLUMN = (
+    # remove duplicates in collection by taking list(dict.fromkeys( collection ))
+    list(
+        dict.fromkeys(
```

Comment:
My understanding is that this only works in py 3.6+, could you use `OrderedDict` instead for compatibility?

Reply:
done